app/styles/components/_grouplist.scss

```diff
@@ -1,21 +1,25 @@
 .grouplist {
   @include span-columns(12 of 12);
-  border-bottom-style:solid;
-  border-left-color:#F7F7F7;
-  border-left-style:solid;
-  border-right-color:#F7F7F7;
-  border-right-style:solid;
-  border-width:0 2px 2px;
+  border-style: solid;
+  border-color: $header-grey;
+  border-width: 0 2px 2px;
   margin-top:0;
-  h2 {
-    width: 100%;
-    font-size: $base-font-size;
-    text-align: left;
-    background-color: $background-grey;
-    padding: $base-font-size;
+    .grouplist-header {
+      background-color: $header-grey; 
+      overflow: auto;
+      padding: .2em;
+      .grouplist-title {
+        float: left;
```

Comment:
Instead of floating these left and right they should be placed on the grid.  See https://github.com/ilios/frontend/blob/master/app/styles/components/_detailview.scss#L46 for an example.  Basically the grouplist already spans 12 of 12 so the title should maybe be 3 of 12 and the h2 9 of 12.